components/organisms/popup/PopupOptions.tsx

```
@@ -0,0 +1,144 @@
+  };
+
+  // 팝업 본문 클릭은 배경 클릭 이벤트로 전파되지 않게 차단한다.
+  const handleDialogClick = (event: MouseEvent<HTMLElement>) => {
+    event.stopPropagation();
+  };
+
+  return (
+    <div
+      className="fixed inset-0 z-50 bg-black/50 flex items-center justify-center"
+      onClick={handleBackdropClick}
+    >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)